docs/devguide/xtypes.rst

```diff
@@ -1364,13 +1505,11 @@ Type System
 ..
     Sect<16.8.1>
 
-* IDL map type
-
-* IDL bitmask type
+* IDL ``map`` type
 
-* .. _xtypes--refheading-toc32438-4273764768:
+* IDL ``bitmask`` type
 
-  Struct and union inheritance
+* Struct and union inheritance
```

Comment:
Unless I'm missing something, I don't think union inheritance is in the specs.

Reply:
It's proposed for future versions https://issues.omg.org/issues/spec/IDL#issue-43778
That distinction was probably missed when this part of the devguide was first authored